tests/by-util/test_basenc.rs

```diff
@@ -32,7 +32,7 @@ fn test_invalid_input() {
     let error_message = if cfg!(windows) {
         "basenc: .: Permission denied\n"
     } else {
-        "basenc: read error: Is a directory\n"
+        "basenc: Is a directory\n"
```

Comment:
A comment might be helpful that we use a different error message than GNU `basenc`.